### PR TITLE
Make changes to Crooks Peak

### LINKED ISSUE #2: Optimise Images
The images in PR #1 are chunky, could they be optimized to be a bit smaller file size? What is the optimum image size based on responsive design, i.e if the page content is never wider than x, image should have a max width of x. Webp might be worth a look but also might be more hassle than it's worth

### LINKED ISSUE #3: Get a cool font
Spec: get a cool font for the website text, something hacky/bitmapy, for example [these fonts](https://www.dafont.com/bitmap.php). 

Requirements:
- free and legal to use, check the liscence
- stored in repo, not from 3rd party hosting
- not massive in size, shouldn't see the font swap mid page load

### LINKED ISSUE #4: Remove text content and seperate sections
PR #1 is adding a lot of text, can it be slimmed down from a wall of text to a shorter game description with a game website link.

Requirements:
- DRY! Remove the wall of text, instead short descrip and link to the game website
- Make the contact part more prominant, perhaps it's own section or inclusion in the main content component or even at the start of the page at the studio introduction. it's the main reason for the page after from the game links tbh

### LINKED ISSUE #5: Jazz up the design 🎷
Make it pop, have a eye catching design that looks a bit different and a nice color scheme / font.

Here are some cool examples / inspirations:
- zainy backghround, very stange [link](http://www.whycjwhy.com/index.html)
- simple but effective, easy to see the games, bio and contacts [link](https://www.danielmullinsgames.com/)
- cool personal profile website [link](https://jgc.org/)
- best game website i've ever seen [link](https://www.inscryption.com/)

Requirements:
- no white or bright backgrounds, they strain my eyes
- can have JS, should work be readable / usable with no JS enabled
- preferably everything on a single page
- still speedy to load, no massive assets

### PR DESCRIPTION
**Pull request by a-jennings**
### Evening Charlie
Decided to have some fun this afternoon and make some changes to your Crooks Peak page. Let me know if you like them!
Page is response, has all the info from the Alien Extraction press pack and I know you're busy actually coding the game ;)

Have fun!

chrome-capture

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,41 +1,74 @@
...
-        .blinking{animation:blinkingText 4.8s infinite}@keyframes blinkingText{0%{color:#000}3%{color:#000}5%{color:transparent}99%{color:transparent}100%{color:#000}}
-    </style>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Crooks Peak | VR Game Studio</title>
+    <link rel="stylesheet" href="css/style.css" type='text/css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
```
Comment (Charlesworth) [inline index.html]: As a performance hound and privacy advocate, I usually like to not use 3rd party hosted fonts, can you either have a local copy of the font (make sure to check font licensing for this) or use system fonts.

File: sass/base/_reset.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  *::after,
+  *::before {
+      margin: 0rem;
+      padding: 0rem;
+      box-sizing: inherit
+  }
+  
+  html {
+      font-size: 62.5%; // 1 rem = 10px; 10/16 = 62.5
+      font-family: 'Lato', sans-serif;
```
Comment (Charlesworth) [inline sass/base/_reset.scss]: Re 3rd party fonts comment

**Review by Charlesworth — CHANGES_REQUESTED** [review]
Looks great, a few change requests:
- If you add an external tool (SASS), add a README.md with the instruction on how to use the  tool (i.e. "$ sass <input.sass> <output.css>" or "$ node run <cmd>")
- Get rid of google fonts
- At some view port sizes (1000x1200) the footer hovers up from the bottom of the page, may be a problem with your "@media (max-width:" stuff
- Can you make the "Available March 2022" a link to "alienextraction.com"

**Comment by Charlesworth** [comment]
@a-jennings didn't like the emoji rain? 😆

**Comment by a-jennings** [comment]
chrome-capture (1)

- New font added to page
- AE-wide image converted to webp:  242k ->  14k
- Use only short game description instead of wall of text
- Change bright color scheme to darker and add animated stars in background (They're much better than the gif shows)

TODO:
-  Background breaks on portrait devises
- Readme for Sass compile

**Review by Charlesworth — APPROVED** [review]
great job, I'll put it live now, cheers buddy

**Pull request merged**